app/Services/ReportbackService.php

```diff
@@ -28,4 +36,37 @@ public function create($data)
 
         return $reportback;
     }
+
+    /*
+     * Handles all the business logic around updating a reportback.
+     *
+     * @param \Rogue\Models\Reportback $reportback
+     * @param array $data
+     *
+     * @return \Rogue\Models\Reportback $reportback.
+     */
+    public function update($reportback, $data)
+    {
+        $reportback = $this->reportbackRepository->update($reportback, $data);
+
+        return $reportback;
+    }
```

Comment:
yea right now this is just a middleman, but see  my comment above, we will need this service to handle all the business logic around creating a reportback (i.e, sending it back to phoenix which @chloealee is working on now, or sending events out to the system that something happened, or emails when stuff gets updated, i dunno) Maybe it is over kill at the moment, but I liked the pattern and felt like it allowed me to wrap my head around the separation of concerns.